### ISSUE TITLE
Debug/Release DescribeZeroCopyTensor linking error

### ISSUE THREAD

**Issue by Sand3r-** [PaddlePaddle/Paddle#15489]
**System information**
-PaddlePaddle version （eg.1.1）or CommitID: `d243e555eb5856d5642706e1aa0d0800b146cae5`
-CPU: including MKLDNN version
-OS Platform: Ubuntu 16.04
-Python version: 2.7.12
- Install method: built from source

**To Reproduce**
```
rm -rf build && mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Debug -DWITH_GPU=OFF -DWITH_MKLDNN=ON -DWITH_TESTING=ON -DWITH_PROFILER=ON -DWITH_STYLE_CHECK=OFF -DWITH_INFERENCE=ON -DWITH_INFERENCE_API_TEST=ON .. && make -j20
```
**Describe your current behavior**
```
[100%] Built target test_analyzer_rnn1
../analysis/passes/libmemory_optim_pass.a(memory_optimize_pass.cc.o): In function `paddle::inference::DescribeZeroCopyTensor(paddle::ZeroCopyTensor const&)':
memory_optimize_pass.cc:(.text+0x1b53): undefined reference to `paddle::ZeroCopyTensor::shape() const'
memory_optimize_pass.cc:(.text+0x1bf9): undefined reference to `paddle::ZeroCopyTensor::lod() const'
memory_optimize_pass.cc:(.text+0x1d17): undefined reference to `float* paddle::ZeroCopyTensor::data<float>(paddle::PaddlePlace*, int*) const'
[100%] Built target test_analyzer_seq_pool1
[100%] Built target check_symbol
[100%] Built target paddle_fluid_origin
Scanning dependencies of target test_inference_recognize_digits_mlp
Scanning dependencies of target test_inference_label_semantic_roles
Scanning dependencies of target test_inference_recognize_digits_conv
Scanning dependencies of target test_inference_image_classification_resnet
Scanning dependencies of target test_inference_image_classification_vgg
Scanning dependencies of target test_inference_recommender_system
Scanning dependencies of target test_inference_word2vec
Scanning dependencies of target test_inference_nlp
Scanning dependencies of target test_train_recognize_digits_conv
Scanning dependencies of target test_train_recognize_digits_mlp
collect2: error: ld returned 1 exit status
paddle/fluid/inference/utils/CMakeFiles/visualizer.dir/build.make:500: recipe for target 'paddle/fluid/inference/utils/visualizer' failed
make[2]: *** [paddle/fluid/inference/utils/visualizer] Error 1
CMakeFiles/Makefile2:60129: recipe for target 'paddle/fluid/inference/utils/CMakeFiles/visualizer.dir/all' failed
make[1]: *** [paddle/fluid/inference/utils/CMakeFiles/visualizer.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

**Other info**
Introduced at some commit after 9252aa41f5af28f73f890b775ce2648c02c45724.

**Comment by luotao1** [comment]
fixed by #15486

**Comment by Sand3r-** [comment]
Thank you so much!
